docs/blog/ibis-to-file.md

```diff
@@ -0,0 +1,56 @@
+# Ibis Sneak Peek: Writing to Files
```

Comment:
Changed from `To_File` to `Writing to Files` since we don't have anything called `to_file`/`To_File` in ibis.